src/App.tsx

```diff
@@ -1,93 +1,188 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useState, useMemo, useEffect, useCallback,
+} from 'react';
+import classNames from 'classnames';
+import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { Header } from './component/Header/Header';
+import { Footer } from './component/Footer/Footer';
+import { TodoApp } from './component/TodoApp/TodoApp';
+import {
+  getTodos,
+  deleteTodo,
+  addTodo,
+  patchTodo,
+} from './api/todos';
+import { NewTodo } from './types/NewTodo';
+import { TodoFilter } from './types/TodoFilter';
+
+const USER_ID = 10610;
+const timeDelay = 3000;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [todoFilter, setTodoFilter] = useState(TodoFilter.ALL);
+  const [isError, setIsError] = useState('');
+  const [query, setQuery] = useState('');
+  const [, setTempTodo] = useState<Todo | null>(null);
+  const areAllTasksCompleted = false;
```

Comment:
The areAllTasksCompleted variable should be a state, not a constant. Change it to use useState.

Reply:
OK, but in the previous comment mentor wrote to delete useState